docs/advanced/sphinx.md

```diff
@@ -125,6 +125,34 @@ The rules should then automatically be applied to your site. In general, these
 CSS and JS files will be loaded *after* others are loaded on your page, so they
 should overwrite pre-existing rules and behavior.
 
+### An example: `superscripted citation`
+
+The default citation style of jupyter book is inline. For example:
+
+```{figure} ../images/inline-citation.png
```

Comment:
Rather than including PNGs of these, could we include actual citations? For the second one, perhaps we could include a specific class for that citation so that we can select on it to make only it superscript?

Reply:
It might be nice, too, to showcase the [other biibliography formats](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#directive-bibliography) sphinx supports like `unsrt`. 

This was something I had a bit of trouble finding at first, and the current documentation only shows `alpha` ! So maybe showing `unsrt` superscripted here would be a nice way to highlight that all of the different styles are available ?

Reply:
@emdupre could explain what unsrt is? I feel like you may have unintentionally just identified yourself as the sphinx expert on bibliographies 😂

Reply:
http://www.cs.stir.ac.uk/~kjt/software/latex/showbst.html although sphinxcontrib-bibtex only supports a few of these by default

Reply:
Although I hope at some point they will also support biblatex, which is the more modern standard: https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/71

Reply:
ahhh interesting - I had no idea. re: `sphinxcontrib-bibtex`, maybe that's something we could upstream?

Reply:
`unsrt` is the style I'm using [here](https://emdupre.github.io/nha2020-nilearn/02-classification-example.html) ! 

It (along with `alpha`, `plain`, and `unsrtalpha` appear to be the only subset of styles currently supported, as @chrisjsewell points out. But for my purposes, it was enough to have `unsrt` available ! So I thought it might be nice to showcase it :smile_cat:

Reply:
@emdupre could you open an issue about what you had to do in order to activate this? We should definitely document it!

Reply:
Great so, I think other citation format docs will be handled in https://jupyterbook.org/content/citations.html#citations-and-bibliographies . For this PR, just need to remove PNGs . @choldgraf @emdupre

Reply:
anything else to take care of here @AakashGfude ?

Reply:
@choldgraf  This PR though is dependent on the implementation of this functionality https://github.com/executablebooks/markdown-it-myst/issues/7 here. So that we can show a text styling similar to the citation style. without actually citing something.

Reply:
should we start targeting that issue?

Reply:
ah yeah, happy to try converging on something there

Reply:
@AakashGfude can we move this PR to a `DRAFT` status and include your update about it being blocked on https://github.com/executablebooks/markdown-it-myst/issues/7 in the top-level comment? Otherwise it's going to look like a stalled PR, when in fact we are waiting on work to be done elsewhere

Reply:
sounds good @choldgraf